packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1020,21 +978,23 @@ contract VotingV2 is
      */
 
     function executeSpamDeletion(uint256 proposalId) external nonReentrant() {
-        require(spamDeletionProposals[proposalId].executed == false, "Proposal already executed");
+        require(spamDeletionProposals[proposalId].executed == false);
         spamDeletionProposals[proposalId].executed = true;
 
         bytes32 identifier = SpamGuardIdentifierLib._constructIdentifier(SafeCast.toUint32(proposalId));
 
         (bool hasPrice, int256 resolutionPrice, ) =
             _getPriceOrError(identifier, spamDeletionProposals[proposalId].requestTime, "");
-        require(hasPrice, "No price found for spam deletion");
+        require(hasPrice);
 
         // If the price is non zero then the spam deletion request was voted up to delete the requests. Execute delete.
         if (resolutionPrice != 0) {
             // Delete the price requests associated with the spam.
             for (uint256 i = 0; i < spamDeletionProposals[proposalId].spamRequestIndices.length; i = unsafe_inc(i)) {
-                uint64 startIndex = uint64(spamDeletionProposals[proposalId].spamRequestIndices[uint256(i)][0]);
-                uint64 endIndex = uint64(spamDeletionProposals[proposalId].spamRequestIndices[uint256(i)][1]);
+                uint64 startIndex =
+                    SafeCast.toUint64(spamDeletionProposals[proposalId].spamRequestIndices[uint256(i)][0]);
+                uint64 endIndex =
+                    SafeCast.toUint64(spamDeletionProposals[proposalId].spamRequestIndices[uint256(i)][1]);
```

Comment:
What is this change? Were these casts unsafe before?

Reply:
yup!

Reply:
Looking at these more closely, I think these should be safe because of line 936 and 937 -- we don't allow these to ever be larger than the current length of the price request array, which should never be more than uint64 max.

Regardless, I don't think checking here is a big deal either way.